### PR TITLE
Add dca option 'customTpl'

### PR DESCRIPTION
**Pull request by OMOSde**

### CODE REVIEW

File: widgets/DcaWizard.php [hunk 1]
```diff
@@ -148,8 +148,10 @@ public function generate()
         $blnShowOperations = $this->showOperations;
         $widget = $this;
 
+        // check for custom template
+        $strTemplate = ($this->customTpl) ?: 'be_widget_dcawizard';
 
-        $objTemplate = new BackendTemplate('be_widget_dcawizard');
+        $objTemplate = new BackendTemplate($strTemplate);
```
Comment (Toflar) [inline widgets/DcaWizard.php:154]: Can just use `$this->customTpl ?: 'be_widget_dcawizard'` right away here, no need for a local variable :)

**Comment by Toflar** [comment]
That looks good now, did you test if that fixes your issue?

File: README.md [hunk 2]
```diff
@@ -61,6 +61,9 @@ This extension provides a widget to handle external table records in the edit mo
         // If this one is not defined, are all listed
         'operations' => array('edit', 'delete'),
 
+        // Use an own template
```
Comment (Toflar) [inline README.md:64]:
```suggestion
        // Use your own custom template
```

Comment (OMOSde) [reply README.md]: Done

**Comment by OMOSde** [comment]
I have tested the new option. Both the deployment of a customized template from a custom bundle, and simply a new template created via the backend.

**Comment by Toflar** [comment]
Thanks! Released as 2.5.0 :)

**Pull request merged**